docs/2.deploy/20.providers/cloudflare.md

```diff
@@ -342,7 +342,7 @@ export default defineNitroConfig({
 
 Next we install the `nitro-cloudflare-dev` module as well as the required `wrangler` package (if not already installed):
 
-:pm-install{name="-D nitro-cloudflare-dev wrangler"}
+:pm-install{name="nitro-cloudflare-dev wrangler -D"}
```

Comment:
We actually have to update this section (both v2 and v3). `nitro-cloudflare-dev` should be replaced with `preset: "cloudflare-"` in Nitro config to leverage built-in dev preset support.